### PR TITLE
Issue 71 limit candidate list

### PR DESCRIPTION
**Pull request by romoy**
Limit the candidate search result set to be 10

### CODE REVIEW

File: api/candidate/candidate.controller.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ exports.searchForCandidate = function (req, res) {
         candidateService
             .searchForCandidate(query.search)
             .then(function (results) {
-                res.send(_.map(results, function (result) {
+                res.send(_.map(results.slice(0,query.limit), function (result) {
```
Comment (mkalish) [inline api/candidate/candidate.controller.js]: I would add or statement here in case there is no query limit

```javascript
query.limit || 5
```

**Comment by mkalish** [comment]
Other than that what change this is looking great!

**Comment by mkalish** [comment]
Conflicts

**Comment by romoy** [comment]
resolved all conflicts @mkalish

**Pull request merged**